packages/database/src/migrations/20210421001607-PsssPalauSyndromBubbleMapOverlay-modifies-data.js

```diff
@@ -0,0 +1,188 @@
+    entityAggregation: {
+      aggregationType: 'SUM_PER_ORG_GROUP',
+      dataSourceEntityType: 'facility',
+    },
+  },
+  presentationOptions: {
+    values: [
+      {
+        color: 'grey',
+        value: null,
+      },
+    ],
+    scaleType: 'neutral',
+    displayType: 'spectrum',
+    measureLevel: 'Facility',
+    defaultTimePeriod: {
+      unit: 'day',
+      offset: 0,
+    },
+    hideByDefault: { null: true },
+    periodGranularity: 'one_day_at_a_time',
+    customLabel,
+  },
+});
+
+const radiusMapOverlayGroup = {
+  id: generateId(),
+  name: 'Syndromic Surveillance Case Numbers (Radius map)',
+  code: 'PW_PSSS_Syndromic_Surveillance_Radius_Map_Country',
+};
+const heatMapOverlayGroup = {
+  id: generateId(),
+  name: 'Syndromic Surveillance Case Numbers (Heat map)',
+  code: 'PW_PSSS_Syndromic_Surveillance_Heat_Map_Country',
+};
+
+exports.up = async function (db) {
+  for (const mapOverlayGroup of [radiusMapOverlayGroup, heatMapOverlayGroup]) {
+    await insertObject(db, 'map_overlay_group', mapOverlayGroup);
+    await insertObject(db, 'map_overlay_group_relation', {
+      id: generateId(),
+      map_overlay_group_id: '5f88d3a361f76a2d3f000004', // Root
+      child_id: mapOverlayGroup.id,
+      child_type: 'mapOverlayGroup',
+    });
+  }
+  for (const [index, { name, dataElement, code, customLabel }] of syndromes.entries()) {
+    const radiusMapOverlay = syndromeRadiusMapOverlay(name, dataElement, code ?? name, customLabel);
+    const heatMapOverlay = syndromeHeatMapOverlay(name, dataElement, code ?? name, customLabel);
+
+    for (const [mapOverlay, mapOverlayGroup] of [
+      [radiusMapOverlay, radiusMapOverlayGroup],
+      [heatMapOverlay, heatMapOverlayGroup],
+    ]) {
+      await insertObject(db, 'mapOverlay', mapOverlay);
+      await insertObject(db, 'map_overlay_group_relation', {
+        id: generateId(),
+        map_overlay_group_id: mapOverlayGroup.id,
+        child_id: mapOverlay.id,
+        child_type: 'mapOverlay',
+        sort_order: index,
+      });
+    }
+  }
+};
+
+exports.down = async function (db) {
+  await db.runSql(
+    `
+      DELETE FROM "map_overlay_group_relation" r
+      USING "map_overlay_group" g 
+      WHERE g.id = r.map_overlay_group_id
+      AND g.code in ('${radiusMapOverlayGroup.code}','${radiusMapOverlayGroup.code}');
+
+      DELETE FROM "map_overlay_group"
+      WHERE code in ('${radiusMapOverlayGroup.code}','${radiusMapOverlayGroup.code}');
+    `,
+  );
+
+  for (const { name, dataElement, code, customLabel } of syndromes) {
+    const radiusMapOverlay = syndromeRadiusMapOverlay(name, dataElement, code ?? name, customLabel);
+    const heatMapOverlay = syndromeHeatMapOverlay(name, dataElement, code ?? name, customLabel);
+    for (const mapOverlay of [radiusMapOverlay, heatMapOverlay]) {
+      await db.runSql(
+        `
+          DELETE FROM "mapOverlay"
+          WHERE id in ('${mapOverlay.id}','${mapOverlay.id}');
```

Comment:
Minor but we're having 2 similar ids in WHERE IN here..?
```suggestion
          WHERE id in ('${mapOverlay.id}');
```